src/Essentials/test/DeviceTests/Tests/Screenshot_Tests.cs

```diff
@@ -11,15 +11,39 @@ public class Screenshot_Tests
 		[Fact]
 		public async Task GetPngScreenshot()
 		{
-			var mediaFile = await Screenshot.CaptureAsync();
+			ScreenshotResult mediaFile = null;
+
+			for (int i = 0; i < 2; i++)
+			{
+				try
+				{
+					mediaFile = await Screenshot.CaptureAsync();
+					break;
+				}
+				catch { }
+				await Task.Delay(1000);
+			}
+
 			var png = await mediaFile.OpenReadAsync(ScreenshotFormat.Png);
 			Assert.True(png.Length > 0);
 		}
 
 		[Fact]
 		public async Task GetJpegScreenshot()
 		{
-			var mediaFile = await Screenshot.CaptureAsync();
+			ScreenshotResult mediaFile = null;
+
+			for (int i = 0; i < 2; i++)
+			{
+				try
+				{
+					mediaFile = await Screenshot.CaptureAsync();
+					break;
+				}
+				catch { }
+				await Task.Delay(1000);
```

Comment:
Maybe we can create some extensions methods in the tests to add some delay passing the time, etc.